.github/workflows/tests.yml

```diff
@@ -28,6 +28,7 @@ jobs:
         pip install -r requirements.txt
```

Comment:
So we don't have to maintain version info both here and in `setup.py`, let's try installing this way:
```
pip install -e .[brotli,cloud-volume,zmesh]
```